File: mssql/scala/create/mssql_create.scala
// -----------------------------------------------------------------
//	create/mssql_create.scala
//
//						Oct/22/2014
// -----------------------------------------------------------------
import scala.io.Source
import java.sql._

// -----------------------------------------------------------------
object mssql_create
{
// -----------------------------------------------------------------
def main (args: scala.Array[String])
{
	println	("*** 開始 ***")

//	val str_connect= "jdbc:sqlserver://host_mssql\\SQLEXPRESS;databaseName=city;"
	val str_connect= "jdbc:sqlserver://host_mssql;databaseName=city;"
	val conn = DriverManager.getConnection (str_connect,"sa","scott_tiger")

	val sss = conn.createStatement ()

	sql_manipulate.drop_table_proc (sss)

	mssql_create_table_proc (sss)

	insert_data_proc (sss)

	sql_manipulate.display_proc (conn)

	sss.close
	conn.close

	println	("*** 終了 ***")
}


// -----------------------------------------------------------------
def mssql_create_table_proc (sss:Statement)
{
sss.execute ("create table cities (id varchar(10), name varchar(20), population int, date_mod datetime)")
}

// -----------------------------------------------------------------
def insert_data_proc (sss:Statement)
{
	sql_manipulate.insert_record_proc (sss,"t1071","前橋",31965,"1998-9-23")
	sql_manipulate.insert_record_proc (sss,"t1072","高崎",29784,"1998-7-16")
	sql_manipulate.insert_record_proc (sss,"t1073","桐生",84162,"1998-5-21")
	sql_manipulate.insert_record_proc (sss,"t1074","沼田",51879,"1998-2-16")
	sql_manipulate.insert_record_proc (sss,"t1075","伊勢崎",65219,"1998-10-14")
	sql_manipulate.insert_record_proc (sss,"t1076","水上",98274,"1998-11-21")
	sql_manipulate.insert_record_proc (sss,"t1077","太田",81563,"1998-12-16")
	sql_manipulate.insert_record_proc (sss,"t1078","安中",47632,"1998-5-8")
	sql_manipulate.insert_record_proc (sss,"t1079","みどり",57943,"1998-7-21")
}

// -----------------------------------------------------------------
}
// -----------------------------------------------------------------

File: mssql/java/create/mssql_create.java
// ---------------------------------------------------------------------
/*
	mssql/java/create/mssql_create.java

					Nov/18/2015

*/
// ---------------------------------------------------------------------
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.Statement;
import java.sql.SQLException;

// ---------------------------------------------------------------------
public class mssql_create
{
// ---------------------------------------------------------------------
public static void main (String args [])
	throws SQLException,
	InstantiationException, IllegalAccessException
{
	System.out.println ("*** 開始 ***");

	try
		{
//		String url = "jdbc:sqlserver://host_mssql\\SQLEXPRESS;";
		String url = "jdbc:sqlserver://host_mssql\\EG;";
		url += "databaseName=city;";

		String user="sa";
		String passwd="Tiger123";

		Connection conn = DriverManager.getConnection (url,user,passwd);

		Statement ss = conn.createStatement ();

		try
			{
			rdb_common.drop_table_proc (ss);
			}
		catch (Exception ee)
			{
			ee.printStackTrace();
			}

		mssql_create_table_proc (ss);
		insert_data_proc (ss);

		ss.close ();
		conn.close();
		}
	catch (Exception ee)
		{
		ee.printStackTrace();
		}

	System.out.println ("*** 終了 ***");
}

// ---------------------------------------------------------------------
static void mssql_create_table_proc (Statement ss) throws SQLException
{
ss.execute ("create table cities (id nvarchar(10), name nvarchar(20), population int, date_mod datetime)");
}

// ---------------------------------------------------------------------
static void insert_data_proc (Statement ss) throws SQLException
{
	rdb_common.insert_record_proc (ss,"t1071","前橋",62375,"2008-4-23");
	rdb_common.insert_record_proc (ss,"t1072","高崎",81463,"2008-8-16");
	rdb_common.insert_record_proc (ss,"t1073","桐生",58172,"2008-5-21");
	rdb_common.insert_record_proc (ss,"t1074","沼田",27953,"2008-9-16");
	rdb_common.insert_record_proc (ss,"t1075","伊勢崎",14259,"2008-10-14");
	rdb_common.insert_record_proc (ss,"t1076","水上",24978,"2008-11-21");
	rdb_common.insert_record_proc (ss,"t1077","太田",89163,"2008-12-16");
	rdb_common.insert_record_proc (ss,"t1078","安中",42357,"2008-5-8");
	rdb_common.insert_record_proc (ss,"t1079","みどり",51783,"2008-7-21");
}

// ---------------------------------------------------------------------
}

// ---------------------------------------------------------------------
